themes/bootstrap3/templates/RecordDriver/DefaultRecord/collection-info.phtml

```diff
@@ -39,18 +39,16 @@
 
     <a id="moreInfoToggle" href="#" class="hidden"><?=$this->transEsc('more_info_toggle')?></a>
     <?php /* Display Main Details */ ?>
-    <?php
-      $formatter = $this->recordDataFormatter($driver);
-      $fields = $formatter->getData($formatter->getDefaults('collection-info'));
+    <?=
+      $this->record($this->driver)->renderTemplate(
+          'core-fields.phtml',
+          [
+            'driver' => $this->driver,
+            'defaults' => 'core',
```

Comment:
Should this be `collection-info` rather than `core`?

Reply:
Absolutely.